app/src/main/java/org/mozilla/focus/components/RelocateService.java

```diff
@@ -125,6 +125,11 @@ private void handleActionMove(final long rowId,
                                   final long downloadId,
                                   @NonNull final File srcFile,
                                   @Nullable final String mediaType) {
+        // if the download id is not in our database, ignore this operation
+        final DownloadInfoManager mgr = DownloadInfoManager.getInstance();
+        if (!mgr.recordExists(downloadId)) {
```

Comment:
If return here does not do anything(such as broadcastRelocateFinished), would it be better if we move this check out to its callee?